docs/plugins/requirements/image-processing.md

```diff
@@ -190,14 +194,13 @@ process is different:
     environmental `PATH` variable. Additionally each library `name` is checked
     in [two variants][find-library-Windows] with the `name` and `name.dll` format.
 
-    The default installation path of [GTK runtime] is:
+    The default binary and shared library path for MinGW-w64 part of [MSYS2] is:
 
     ```powershell
-    C:\Program Files\GTK3-Runtime Win64
+    C:\msys64\mingw64\bin
     ```
 
-    and the libraries are in the `<INSTALL-DIR>\lib` directory. Use the debug
-    script below to check if the path is included. If it isn't then:
+    Use the debug script below to check if the path is included. If it isn't then:
 
     1. Press ++windows+r++.
     2. Run the `SystemPropertiesAdvanced` applet.
```

Comment:
Are libs installed via pacman available outside of the MSYS2 terminal? 
Also if they should be, then some path is added to the PATH variable, are those instructions still accurate?

Reply:
Shared libraries are in `\bin` directory, which is noted in the changed lines just above this one; `\lib` directory contains only static libraries, which should not be in `PATH`, as far as I understand.

Reply:
So the 4th point in list should point to the `bin` and not `lib` directory, right?

Reply:
Good catch. Let me clear that up.